---
author: John Micthell
layout: post
page_id: 1350
title: Molecular Mapping Workshop Abstracts 2006
---
All abstracts from the [2nd Molecular Mapping Workshop](2006/10/9 "wikilink") held in Palmerston North on October 9-10 are available here. \_\_TOC\_\_

### Keynote: Mapping complex disease traits in human

**Grant Montgomery, Queensland Institute of Medical Research.**

(No abstract).

### Towards cloning of the genes underlying a major pea (Pisum sativum) QTL determining components of yield

**Gail Timmerman-Vaughan (1), Sarah Murray (1), Linda Falloon (1), Tonya Frew (1), Clarice Coyne (2)**

1 Crop & Food Research, PO Box 4704, Christchurch, New Zealand

2 Western Regional Plant Introduction Station, USDA-ARS, 59 Johnson Hall, Washington State University, Pullman WA, USA

Yield per se and determinants of yield are arguably the most valuable traits for grain crops. Insights into the identity of genes controlling yield determinants will lead to an enhanced understanding of the pathways and processes that control seed yield. In previous work, we identified a major QTL determining the trade-off between two yield determinants on linkage group I of pea. Research is now underway to identify candidate genes associated with this QTL. A well saturated map of the genomic region of interest has been developed, the sequences of AFLP and other markers in the QTL region have been determined, and orthologous or highly similar sequences from the model legume Medicago truncatula have been identified. Using genomic sequence data from the international M. truncatula sequencing effort, BAC contigs containing sequences from the QTL region have been identified. Further markers have been developed by converting M. truncatula sequences into polymorphic PCR-based markers and placing these on pea linkage maps. Current efforts are focussed on applying an association mapping strategy to narrow down the genomic region determining this QTL and therefore to identify candidate genes. To date this strategy has provided promising results, and a sequence that is in linkage disequilibrium with the QTL has been identified.

### Construction of a dense bovine SNP linkage map

**Juan Arias (1), Wouter Coppeiters (2), Paul Fisher, Michael Keehan, Richard Spelman.**

1 Livestock Improvement Corporation, Hamilton, New Zealand

2 University of Liege, Belgium

We have constructed linkage maps for 29 autosomal bovine chromosomes. Briefly, the Affymetrix GeneChip® Bovine Mapping 10K SNP Kit was used to genotype over 1600 crossbred Holstein-Friesian x Jersey cows and their sires. SNPs were initially assigned to a chromosome strictly based comparative alignment between the human genome and a dense bovine genetic map. A SNP showing any of the following criteria: departure from Hardy-Weinberg, a minor allele frequency lower than 5% or non-inheritance was deleted from further analysis. The software MultiMap was utilized for linkage map making. For each chromosome, three initial low density linkage maps were made with a restraint of a minimal LOD score equals to three. One produced by MultiMap itself; another which also contained a small frame consisting of a linkage map of microsatellites previously mapped using genotypes from same animals and last, a linkage map produced by the CRI-MAP, BUILD option. These three linkage maps were combined maintaining order within each linkage map and served as frame in an iterative MultiMap linkage map making procedure but with a LOD score of two. During this phase, several SNPs initially assigned to a specific chromosome show clearly no linkage to other SNPs or microsatellites. Approximate 80% of these SNPs were reassigned to another chromosome by performing a CRI-MAP, TWOPOINT analysis against 291 microsatellites covering the complete bovine genome. Then, another phase of MultiMap linkage making was undertake with newly assigned SNPs and rest of unmapped SNPs. 57.11% of initial SNPs have been mapped to unique positions. The average distance between mapped SNPs is at 0.69 cM. The 29 autosomal linkage maps cover over 3000 cM.

### Experimental design and statistical methods for QTL mapping, association studies, and candidate gene detection

**Rod Ball**

Ensis Wood and Fibre Quality, Scion, Private Bag 3020 Rotorua

We give an overview of statistical work for gene mapping in the Scion Cell Wall Biotechnology Centre, including statistical methods for QTL mapping, association mapping and candidate genes. Recent work on combining QTL and association studies, and quantifying evidence for candidate genes co-locating with QTL is discussed in more detail.

Association studies utilise historical linkage disequilibrium. In principle, with 500,000 or more SNP markers QTL, genes can be located to within 6kb. However this achieving this high resolution requires genotyping many markers per individual, for large sample sizes. Combining association and QTL mapping can improve the posterior odds while reducing total genotyping.

Candidate polymorphisms mapping close to QTL have increased probabilities to represent functional loci. Evidence for candidate genes co-locating with QTL is quantified in terms of prior odds for the candidate, a Bayes factor, Bc, representing sequence specific evidence for the candidate, and a Bayes factor, BQ(x), derived from QTL mapping data, where x is the candidate's map position. Additionally we allow for uncertainty in the estimated map position.

### Mapping the gene for inherited cataract in sheep

**Gareth Wilson**

Lincoln University

Lincoln University maintains a flock of sheep which spontaneously develop cataracts within two months of birth. The cataract is inherited as an autosomal dominant trait and is being used as a model for human cataract in trials of drugs developed to prevent cataract formation. Determining which gene is responsible for the cataract would help to determine the mechanism of cataract formation and the relevance to human cataracts.

In order to locate the cataract gene on the sheep genome, a pedigree of about 300 animals was assembled, consisting of four cataract rams, their offspring with normal ewes, and some of their ancestors. All offspring were scored for cataracts using slit lamp microscopy at two months of age. DNA extracted from the animals was subjected to a multiple stage genome scan, testing widely separated microsatellite markers throughout the genome and then testing more narrowly spaced markers in the linked region.

After testing 149 markers on a subset of the pedigree, and analysing the results with the genetic linkage program CRIMAP, significant linkage with LOD scores greater than 3 was found to markers on sheep chromosome 6. More markers were tested on chromosome 6, using the entire pedigree. One new marker had a LOD score of 38.71, indicating very strong linkage to the cataract gene. The data from the high resolution scan was used in a multipoint linkage diagram to narrow down the location of the cataract locus to between 123 and 130 cM from the centromere.

### Kiwifruit polyploid fingerprinting

**Tim Lawrence**

HortResearch, Mt Albert, Auckland

The project aims to validate a method to statistically support fingerprinting data produced from polyploid varieties using kiwifruit as our initial system. This is beneficial for commercial clients as it provides a statistical value for use in the defence of a variety rights court case, previously only available in diploid cases. This technology will also be advantageous in quality control systems of polyploid varieties in breeding programmes and growing orchards.

A statistical method, PAFEst (polyploid allele frequency estimator) based on an EM (expectation maximisation) algorithm has been proposed by De Silva et al. 2005 for estimation of the frequency of alleles in polyploids. The effect of auto and allopolyploidism, number and frequency of alleles, null alleles, sample size and the degree of “inbreeding” (in an allelic sense) were all taken into account in developing the methodology. The same authors have further investigated the effects of preferential pairing and double reduction on the allele frequency estimates (De Silva et al, 2006).

This project aims to validate this method through independently measuring the dosage of alleles by a two pronged approach, one from the marker data itself using peak height and area in the case of SSR markers that behaved in the most dosage dependent manner, and the second using molecular methods to clone and sequence the alleles present. The results of these two methods were compared to the predicted scores from computationally derived data.

### Combination of pedigree recording and SNP genotypes to improve bovine parentage matching

**Paul Fisher (1), Ben Malthus (2), Mark Walker (1), Geoff Corbett (3), Richard Spelman (1)**

1 Livestock Improvement, Hamilton

2 ZyGEM Corporation Limited

3 Waikato Innovation Park

Currently a 14-microsatellite panel is being used to parentage test New Zealand dairy animals. New SNP-based platforms offer operational advantages over the conventional microsatellite-based ones making them a promising alternative for parentage exclusion. We used both simulated and empirical data to ascertain the number of SNPs that would provide a power of exclusion that is comparable to the current microsatellite panel. Although the exact number was dependent on several factors, a 40-SNP panel (where the minor allele frequency was 0.35 on average) was always better at assigning parentage than the 14-microsatellite set. However, it can be difficult to attain 100% exclusion rates in dairy herds with genotyping data alone. This is because several sires and hundreds of dams could be potential parents of the tested daughters. Therefore software was written to combine mating records and birth date/calving date alignments with the genotyping data. Using a real herd, inclusion of these records increased the power of exclusion dramatically for both SSRs (from 72% to 95%) and SNPs (from 74% to 100% for the 40 SNP panel). This highlights the importance of combining genotypes with on-farm data to maximise the ability to assign parentage in the New Zealand dairy herd.

### Mapping in forages: from QTL discovery to MAS

**Marty Faville**

AgResearch, Palmerston North

AgResearch is engaged in the development of marker-assisted selection (MAS) to support the production of superior white clover and perennial ryegrass varieties through plant breeding. Traits targeted for improvement reflect stakeholder interests, and include forage yield (FY), seed yield (SY), endophyte compatibility, forage nutritive value, abiotic stress tolerance and plant persistence. Discovery programmes are primarily QTL-based, leveraging off five genetic linkage maps (two ryegrass, three clover) developed in F1 full-sib families with SSR markers sourced from proprietary EST (ryegrass and clover) or GeneThresher™ (clover) DNA sequence databases. Analysis of glasshouse and field data has identified numerous, moderate-effect QTL for FY in ryegrass, and facilitated dissection of the influence of FY component traits at these loci. Marker candidates from FY QTL have undergone preliminary validation screening, with solid prospects identified for further evaluation. Seed yield DNA tests for routine use in white clover breeding programmes have been developed from major QTL for SY components and SY per se, now confirmed in independent mapping populations. Multi-year validation studies for DNA tests in commercial, multi-parent synthetic populations have shown \>50% of tested populations have a significant relationship between SY performance and marker test state, while experimental synthetics based on marker selections are currently under evaluation. Comparative genome relationships to model species (rice for ryegrass; Medicago truncatula for clover) have been successfully exploited for templating targeted marker enrichment at QTL, and for candidate gene-based marker development, in both species. New research includes initiation of a collaborative association mapping study in ryegrass, and a project that integrates high-throughput metabolite profiling with QTL analysis.

### Use of stochastic simulations to investigate the power and design of using SNP Chips for whole genome association studies in farm animals: a small example

**Benoit Auvray**

AgResearch, Invermay

Whole genome SNP chip technologies offer opportunities for gene discovery in farm animals. However, the use of those chips requires careful planning due to technological, cost or time related constraints inherent to association studies or other linkage disequilibrium-based methods. These constraints depend on many factors specific to each experiment. It is therefore very desirable to maximise beforehand the design of an experiment and to have a good grasp on how powerful this design is given the resource on hand, before conducting any large experiment using SNP chip technologies.

Stochastic simulations of populations of animals, using a mixture of observed and assumed starting parameters for the modelling of the population structure and history and the QTL characteristics and effects can be used to address this problem. Indeed, extensive simulation of populations with a range of different starting assumptions, the virtual genotyping of different selected animals with different genotyping technologies and their subsequent analysis with different statistical methods should allow the identification of the most suitable genotyping strategy and method of analysis.

A small example of such a simulation is presented here.

### Molecular breeding and candidate gene mapping in forest trees - an update

**P.L. Wilcox (1), R.D. Ball (2), E.J. Telfer (1), S.A. Cato (1), J. Douglas (1), S. Capper (1), R.D. Burdon (3)**

1 Cellwall Biotechnology Centre, Scion, Rotorua

2 Ensis Wood and Fibre Quality, Scion, Rotorua

3 Ensis Genetics, Scion, Rotorua

Gene mapping and related activities at Scion are centred around two areas: development and applications of tools for breeding purposes, and assisting molecular biologists to identify specific genes for subsequent manipulation of cell wall properties. The major emphasis to date has been on the latter. Here, we have mapped 113 expressed genes, with 27 collocating with QTL for wood density (WD), a key wood-quality trait. We have also found evidence for co-location of wood property QTL among three species: P. radiata, P. pinaster and P. taeda. This has allowed us to substantively increase the number of candidate genes collocating with QTL for a range of wood properties. We have also undertaken association genetics in Pinus radiata, and have found associations between polymorphisms in a dehydrin gene (Dhn1) with WD in a large association population (n = 1950). This gene had collocated with WD QTL, and appears to be under strong balancing selection. These results conceptually demonstrate the applicability of association genetics in forest genetics for both high-resolution candidate gene mapping as well as breeding applications. These projects are complemented by development of Bayesian methods for both experimental design and analyses of results. Recent efforts have included combining QTL and association mapping to improve posterior odds while reducing genotyping, as well as developing methods to quantify evidence for associations between candidate genes and traits that account for prior information on specific candidate genes. Our other focus is on application of molecular markers for operational breeding purposes, largely for QA/QC. SSR markers have been developed not only for P. radiata, but also Eucalyptus fastigata and E. nitens. These tools have recently been evaluated by scientists from Ensis Genetics for pedigree reconstruction in operational breeding experiments as an alternative to conventional testing.

### Genetic variation of key serotonergic transcription factor genes

**Martin Kennedy**

University of Otago

Dysregulation of the serotonin neurotransmitter system has long been known as an important factor in the pathophysiology of depression. Recently, knockout mice for various geneshave identified the key transcription factors (NKX2, GATA2, LMX1B, FEV) required for development of the serotonergic neuron. We have identified common genetic variants in human NKX2, GATA2, LMX1B and FEV. Polymorphism detection was achieved using a combination of DHPLC and sequencing in 22 individuals from a family study of depression. Although no coding SNPs were detected in any genes, several promoter polymorphisms were identified. In particular, GATA2 contained eight polymorphic sites (represented as six haplotypes) within 1.4Kb of the transcriptional start site. These haplotypes altered basal promoter activities in luciferase reporter assays performed in COS-7 (primate kidney) and RN46A (rat serotonergic) cells, in one case reducing expression up to 40% (P \< 0.01). Interestingly, the impact of these polymorphisms on promoter function is not consistent between COS-7 and RN46A suggesting cell-line specific effects on gene regulation. In comparison, a single SNP found in the NKX2 promoter reduced expression by 30-40% in both cell lines (P \< 0.01). Given the apparent functional effects of these variants, genotyping methods are being developed (allele-specific PCR and SNaPshot) for their detection in a large cohort of depressed individuals. Genetic association studies will be carried out to evaluate the potential relevance of such variants to clinical phenotypes and treatment outcomes in depression.

### Construction of a saturated genetic linkage map for apple rootstocks

**Jean-Marc Celton (1), Stuart Tustin (2), Erik Rikkerink (3), David Chagné (1), Barbara Ambrose (4), Ian Ferguson (3) and Susan Gardiner (1)**

1 HortResearch, Palmerston North

2 HortResearch, Havelock North

3 HortResearch, Auckland

4 Massey University, Palmerston North

Marker assisted selection (MAS) for apple (Malus x domestica Borkh.) offers quick and reliable prediction of the phenotype of a young seedling and is opening new ways to breeders. Genetic mapping in segregating populations enables the discovery of genetic markers linked to desired traits. We constructed a linkage map developed from a cross between ‘M9’ and ‘R5’ apple rootstocks using 194 SSR (Simple Sequence Repeats) markers, 19 SCARs, 14 SNPs and 43 RAPDs. A new set of 42 polymorphic SSRs was developed from apple EST sequences and mapped throughout the genome. Markers were derived from EST sequences containing di-, tri- and tetra-nucleotide repeats. All 17 linkage groups have been identified and aligned to existing maps using markers in common. Each linkage group carries a minimum of three SSR markers. The two parental maps are the first genetic map constructed for apple rootstock and span 1169.3 cM and 1083 cM respectively, which is comparable with the latest apple genetic maps. These linkage maps will help elucidate the genetic basis of architectural characters such as the dwarfing effect induced by the apple rootstock ‘M9’, the analysis of disease resistance traits such as fire blight (Erwinia amylovora), and the study of the resistance to insects, such as woolly apple aphid (Eriosoma lanigerum). This will ultimately lead to the development of new markers that will be directly available to apple breeders, and help to identify the causative genes by map-based cloning.

### Two dimensional genome scans

**Ken Dodds**

AgResearch, Invermay

The standard approach to analyzing a genome-scan QTL experiment involves testing for an effect due to the grand-parental origin of the alleles at each position in the genome. A two dimensional scan (performing a test at each pair of positions in the genome) is able to detect epistatic QTL, including those whose marginal effects are zero or too small to be detected using the one dimensional scan. This talk will discuss a number of aspects in relation to undertaking a two dimensional scan.

### Haplotype discovery in candidate genes for increasing forage biomass

**Geoff Gill**

Pastoral Genomics / ViaLactia Biosciences, Auckland

SNP and haplotype discovery is an important prerequisite of candidate gene-based association studies. To efficiently survey haplotype diversity of a gene in a large population, haplotype libraries were generated by cloning PCR amplicons derived from DNA pools. Sequencing of cloned haploid genomic DNA not only provided haplotype information but also avoided the problem of high frequency indels disrupting base calling as occurs when sequencing diploid DNA. This method was evaluated on two candidate genes targeted at increasing forage biomass and are currently undergoing functional testing in perennial ryegrass. Suppression of both genes is desirable and so naturally occurring mutations in these genes that knockout or reduce protein activity are of particular interest. Key sequence variants will be tested for association with phenotypic variation in genetically diverse natural populations and appropriate pair-crosses. Results from haplotype discovery, mutation screening and estimation of linkage disequilibrium will be discussed.

### Pastoral Genomics: clovers and inter-specific QTL discovery

**Griffiths AG (1), Williams WM (1), Hay MJM (1), Hancock KR (1), Scott AG (1), Ellison NE (1), Collette VE (1), Richardson KA (1), Jahufer MZZ (1), Baird IJ (1), Cooper BM (1), Rasmussen S (1), Woodfield DR (1), Crush JR (1), Ouyang L (1), Nichols SN (1), Anderson CB (1), Lawn JR (1), Dunn A (1), Ford JL (1), Allan AM (1), Franzmayr BK (1), Greig MN (1), Maher DA (1), Tilbrook JC (1), Williamson ML (1), Warren JWG (1), Khan AK (1), Bickerstaff P (2), Jones CG (2), Jones CS (1)**

1 AgResearch Ltd, c/o Grasslands Research Centre, Palmerston North, New Zealand

2 Pastoral Genomics, c/o ViaLactia Biosciences (NZ) Ltd, Auckland, New Zealand

White clover (Trifolium repens) is a source of both high quality forage and plant-available nitrogen, and is integral to sustainable temperate pastoral agriculture systems. The Pastoral Genomics consortium has established an interdisciplinary programme combining functional genomics, metabolomics, genetic linkage mapping, quantitative trait loci (QTL) analysis and plant breeding. This has allowed Pastoral Genomics to develop a significant toolset that includes:

-   Sequence databases (GeneThresher® and Expressed Sequence Tag databases)
-   Germplasm developed for QTL identification and cultivar production
-   Characterised genes and promoters
-   Optimised plant transformation systems
-   Microsatellite and single nucleotide polymorphism marker resources
-   Genetic linkage maps of white clover (allotetraploid) and its diploid relative T. occidentale
-   Bioinformatic resources

These tools are being used to exploit the genomes of white clover and its near relatives for enhanced agronomic production through the application of marker-assisted selection to conventional breeding. QTL discovery in both white clover and T. occidentale have identified co-located QTLs for the same trait, and several examples where significant QTLs in one species were mirrored by strongly suggestive QTL in a similar location in the other species. These inter-specific QTL comparisons have provided a useful tool for interpretation of QTL data, particularly for sub-significant but strongly suggestive QTLs, and enhance dissection and exploitation of clover traits for the advancement of pastoral agriculture.

### Efficient multilocus association mapping using clustered haplotypes

**Sharon Browning**

University of Auckland

I will describe a new statistical method for multilocus analysis of association between genetic markers and a trait (e.g. case/control status) in unrelated individuals. The method has several important advantages over existing methods. It automatically adapts to the degree of linkage disequilibrium in a region, balancing the information content per haplotype cluster with the number of haplotype clusters, to achieve high statistical power. It is computationally efficient, and can be used with whole genome association data (hundreds of thousands of markers and thousands of cases and controls), with computing time taking only a day or two, including permutation testing to adjust for multiple testing. Simulation results show that the method is more powerful than single-marker tests for detecting low-frequency disease-susceptibility variants.

### Comparative mapping in Pinus and collocation of wood QTL

**Emily Telfer (1), C. Dana Nelson (2) , Craig S. Echt (2) , Phillip L. Wilcox (1)**

1 Scion, Rotorua, New Zealand

2 Southern Institute of Forest Genetics, USDA Forest Service, MS 39574, USA

Comparative mapping among highly syntenic species, such as in the genus Pinus, affords the opportunity for transposing gene location onto maps thus eliminating the need to map orthologues. This is of particular benefit for identification of potential candidates that collocate with QTL. It potentially increases the power to detect QTL for multi gene traits, such as those associated with wood quality. To date we have undertaken comparative mapping in Pinus radiata, P. taeda and P. pinaster, aligning 75 markers common to at least two of three the species, including 28 SSR loci and 47 EST loci. In addition, 13 markers were common to all three species. By comparing linkage maps of P. radiata, P. taeda and P. pinaster we are identifying collocalising QTL for wood density, microfibril angle and cell wall chemistry. In a preliminary study comparing the co-occurrence of wood density QTL in P. radiata and P. taeda, we found that observed co-location of QTL was highly unlikely to be attributed to chance alone (P = 0.002). The inclusion of multiple pine species with sufficient common markers should enable us to produce a robust combined Pinus linkage map, with which any number of additional QTL can be compared. From this we will be able to identify potential candidate genes, associated with traits of interest, for further studies, including association genetics and functional analysis.

### Quantifying population admixture in breeds of livestock a potential confounding effect in genetic association studies

**Richard Hall**

AgResearch, Invermay

The distinction between breeds of livestock, such as those found in cattle, has usually been evaluated by phenotypic characteristics such as coat colour or the presence or absence of horns. Methods that utilise genetic markers in the taxonomic classification of species, have also been applied to within-species analysis and the genetic distinctions between various breeds of livestock have been characterised. One such method that has recently been developed, called STRUCTURE, allows for the determination of the probable ancestry components of an individual animal. This method was applied to genotype data from four commercially important cattle breeds from the New Zealand cattle herd. A panel of 34 microsatellite markers was clearly able to resolve the breed groups of Angus, Hereford, Friesian and Jersey. A detailed study of the New Zealand Angus population using a combination of 29 SNPs and the entire microsatellite panel failed to reveal any evidence for population sub-structure in New Zealand Angus sires, indicating that this population is relatively homogeneous. The utility of STRUCTURE in a hybrid cattle population was also assessed. Taken together, the results of this study indicate that cattle breeds can be distinguished based upon genotype alone and that individual hybrid animals ancestry can be estimated with a reasonable degree of accuracy. These data will be of benefit to association studies in cattle, as a method for assessing the presence of admixture, which is a major confounding factor when testing for genetic association.

### Sequencing the ovine genome on a (small) budget

**John McEwan**

AgResearch, Invermay, Mosgiel

All the key technologies now being developed in mammalian genomics assume the genome sequence of the species is available. Perhaps the most recent and most critical to locating variants of economic interest are highly multiplexed “SNP chips”. Species are rapidly being segregated in those with and without a genome sequence with NIH and USDA now not funding any genomic research in species that have less than 6X genome sequence coverage. While sequencing costs have fallen dramatically over the last 15 years there is still a 10 fold gap between likely funding and the cost of current approaches. The International Sheep Genome Consortia is developing a coordinated stepwise approach using a variety of novel methods which at will deliver a well ordered rough draft of the 3Gbp ovine genome and several hundred thousand SNPs for less than \$5M. The strategy and progress to date which has already produced a high quality physical map of the ovine genome will be outlined.

### Genetic determinism and QTL mapping of plant parameters involved in the efficient and sustainable utilisation of forage marze in animal nutrition

**Lethbridge, L. A (1), Margerison, J. K. (1), C. S. Brennan (1), Hentenyi, L. (2) Chrenkova, M G. (2) and Hastings (3)**

1 Institute of Food Nutrition and Human Health, Massey University

2 RIAP, Slovakia

3 School of Biological Sciences, University of Plymouth, UK

The objective was to measure forage maize (M) nutrient composition (DM, CP, NDF, ADF, ADL, starch, water soluble carbohydrates (WSC), ash) and degradability (dg) to identify qualitative trait loci (QTL) using mapping techniques and to use these to improve forage quality. Near isogenic lines (NILs) of FM (n=350) were produced from 3 parent varieties and grown in randomised block design in 3 EU countries. In the UK M was harvested in mid October (27 % DM). Data from parental varieties (ANT, NKB) with 8 NILs (A to H) (in g/kg DM unless stated otherwise) were analysed (ANOVA) and compared (Tukey’s test) to assess variability. DM: A 281.1, B 284.8, C 250.2, D 299.5, E 292.9, F 288.5, NKB 270.9, ANT 294.5, G 295.8, H 284.6 (sed 2.11, P\<0.001). NDF: A 483.3, B 485.7, C 490.7, D 455.7, E 455.3, F 398.3, NKB 489.7, ANT 410.3, G 471.3, H 452.3 (sed 31.73). ADF, A 289.7, B 271.33, C 320.0, D 253.3, E 238.3, F 307.3, NKB 275.7, ANT 241.7, G 255.0, H 311.3 (sed 16.56, P,0.05). ADL: A 20.0, B 19.3, C 46.7, D 26.7, E 32.0, F 38.7, NKB 44.3, ANT 13.3, G 32.3, H 33.3 (sed 16.06). Ash: A 42.9, B 39.3, C 53.1, D 31.4, E 39.4, F 37.6 NKB 43.0, ANT 39.9, G 37.9, H 115.9 (sed 5.15, P\<0.001). CP: A 73.5, B 63.0, C 72.1, D 67.7, E 57.7, F 57.5, NKB 59.2, ANT 63.5, G 55.3, H 59.2 (sed 1.22, P\<0.001). Starch: A 159.5, B 247.7, C 98.3, D 201.8, E 507.0, F 324.0, NKB 122.0, ANT 434.9, G 464.9, H 148.7 (sed 88.69, P\<0.001). WSC (mg/ml DM): A 5.00, B 13.3, C 15.0, D 10.0, E 9.8, F 5.0, NKB 2.9, ANT 8.1, G 4.1, H 17.8 (1.30, P\<0.001). M dg; quickly (a), slowly (b) and total dg rates, starch development, structure and relationship with protein varied significantly between NILs, pectin and amylopectin did not. Starch levels increased as DM increased, while NDF, ADF, ADL, CP and ash decreased. Genotype affected starch, ash and crude protein levels, starch and fibre degradability, starch structure and relationship with protein (P\<0.001), but did not affect pectin and amylopectin. Distinct QTL were involved in M composition, degradability and starch characteristics. QTL mapping can be used in genetic selection and breeding of M to increase forage intake, utilization efficiency and agricultural sustainability.

### Levels and patterns of linkage disequilibrium in the New Zealand dairy population

**Richard Spelman, Mike Keehan and Juan Arias.**

Livestock Improvement Corporation, Hamilton

A F2 experiment between the two predominant breeds in the New Zealand dairy industry has been undertaken to identify the genetic differences between the two breeds. Eight hundred F2 female progeny were generated from 6 F1 sires and 800 F1 dams. This pedigree, with the addition of the F0 sires, has been genotyped with a 10,000 SNP panel. Estimates of the level and pattern of linkage disequilibrium in the two parental populations have been undertaken through the derivation of F0 maternal haplotypes. Linkage disequilibrium, measured by r2, decays to levels with below 0.05 when markers are ½ cM or greater apart in distance. This level of LD is less than estimated from earlier studies. The level of LD in both populations is similar as is the patterns of LD over the chromosomes. This may be indicative of selection sweeps in both populations where they have been selected under a common selection objective.

### Comparison of Framework Mapping using JoinMap and Outmap

**H.N. De Silva, L.G. Fraser, G. Tsang, M.A. McNeilage and P. Datson**

HortResearch, New Zealand

There are three key steps in constructing a framework map of molecular markers: calculating pairwise recombination fractions, establishing the linkage groups, and finally ordering of markers within a group and estimating their map distances. Genetic maps derived from recombination data are only an estimate of the true grouping and distance between markers. Statistically speaking the precision and accuracy of the map will depend on factors such as marker informativeness, family size, number of recombination events between markers, and the statistical methodology used in map construction.

In the last few years HortResearch has embarked on a project to construct a framework map of the diploid Actinidia chinensis [1], a study based on a full-sib family. Data have been collected on \~ 600 markers, which are mainly EST-derived microsatellites. A preliminary map constructed has included \~60% of the markers in 35 or so linkage groups. In this presentation, we will consider some of the issues where computerised procedures are used to construct framework maps. We will make initial comparisons of male and female maps and investigate how they relate to the consensus map, using selected linkage groups. Two computer programmes used in linkage mapping are JoinMap and Outmap. We will compare the statistical methodology used in the two, and compare the resulting maps in relation to the same selected linkage groups.

[1] Fraser LG, Harvey CF, Crowhurst RN and De Silva HN (2004). EST-derived microsatellites from Actinidia species and their potential for mapping. Theo Appl Genet 108: 1010 – 1016.

### Introduction to MapNet Wiki

**Anar Khan**

AgResearch, Invermay

MapNet is a network of scientists undertaking research in gene mapping and related molecular marker technologies in New Zealand, with primary aims being to co-ordinate scientists in disparate sectors and to foster the development of multidisciplinary and cross-sector teams. We have implemented a Wiki, a website based on the MediaWiki software package, to provide a readily-accessible portal for information exchange and communication for MapNet participants. We will discuss the general principles of what a Wiki is and how it is used. This will be followed by a case-study detailing MapNet's requirements for a collaborative website, and how these requirements have been met with MediaWiki.

Click [here](Media:Introduction to MapNet Wiki.ppt "wikilink") to download the presentation.

Click [here](Media:MapNet_Info_Sheet.doc "wikilink") to download the MapNet Info Sheet.

